<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgTimespanFilterWidget</class>
 <widget class="QWidget" name="MsgTimespanFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="tmstmpGrb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Message Timestamp</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="fromLbl">
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="fromHorzL">
        <item>
         <widget class="QDateTimeEdit" name="filterTimeStampFromDateTimeEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumDate">
           <date>
            <year>7999</year>
            <month>12</month>
            <day>31</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>dd.MM.yyyy HH:mm:ss.zzz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="enableTimestampFromFilterCheckBox">
          <property name="toolTip">
           <string>Enable the Message Timestamp From Filter</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>Enabled</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="fromHrzSpcr">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="toLbl">
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="toHorzL">
        <item>
         <widget class="QDateTimeEdit" name="filterTimeStampToDateTimeEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="displayFormat">
           <string>dd.MM.yyyy HH:mm:ss.zzz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="enableTimestampToFilterCheckBox">
          <property name="toolTip">
           <string>Enable the Message Timestamp To Filter</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>Enabled</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="toHorzSpcr">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="9" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="globalEnableTimestampFilterCheckBox">
          <property name="toolTip">
           <string>Enable the Message Timestamp Filter (the individual Filters still need to be enabled)</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>Global Enabled</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="filterTimestampLoadBtn">
          <property name="toolTip">
           <string>Open a Message Timestamp Filter configuration (*.json)</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filterTimestampSaveBtn">
          <property name="toolTip">
           <string>Save the current Message Timestamp Filter Configuration (*.json)</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="invertTimestampFilterCheckBox">
          <property name="toolTip">
           <string>Invert the Message Timestamp Filter</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="text">
           <string>Inverted</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
